unlock-js/src/__tests__/integration/walletServiceIntegration.test.js

```diff
@@ -138,7 +139,111 @@ describe('Wallet Service Integration', () => {
         })
       })
 
-      describe('purchaseKey', () => {})
+      describe('purchaseKey', () => {
+        let tokenId
+        let key
+        let owner, sender
```

Comment:
Can we make these names a bit more specific? I wasn't sure what these roles were until I'd read the tests a few times. Maybe something like `keyPurchaser` and `keyRecipient`?

Reply:
yes!